ystia/yorc
# Tech/gh 613 allow to retrieve error for task

# Pull Request description

## Description of the change

- Create task error message filled with raised error during task execution
- Blank error message when failed workflow is resumed

### How to verify it
- For task query, see internal PR
- For workflow task, deploy the following app on the location you want with A4C:
[ErrApp_0.1.0-SNAPSHOT.zip](https://github.com/ystia/yorc/files/4344201/ErrApp_0.1.0-SNAPSHOT.zip)

It contains an error component that generate an error
You need to import https://github.com/ystia/tosca-samples/tree/develop/org/ystia/yorc/samples/tests/errors nefore add the topology template.
When you get task info from yorc CLI, you should get:

$ yorc d tasks info ErrApp-Environment <TASK_ID> -w
Task:  e202a35f-e98d-474b-81b2-30398f44ca12
Task status: FAILED
Task type: Deploy
Task Error Message: exit status 2


### Description for the changelog

## Applicable Issues
fixes #613
